src/delay.rs

```diff
@@ -18,24 +18,26 @@ use std::time::Duration;
 use embedded_hal::blocking::delay;
 
 /// A `Delay` implementation that does not actually block.
-pub struct MockNoop;
+pub struct NoopDelay;
+/// Deprecated name for no-op `Delay` implementation.
+pub type MockNoop = NoopDelay;
```

Comment:
Hmm, I just realized that a type alias can't be used as constructor. Thus, `let m = MockNoop;` will not compile.

Instead, let's just remove `MockNoop`, breaking changes are OK. However, please add a note to `CHANGELOG.md` about the rename.